### PR TITLE
add license and copyright notices

### PR DESCRIPTION
**Pull request by ElenaKusevska**
@gocemitevski Hi Goce, could you please review this PR and if it's OK merge it. We have to add licenses and copyright notices to all Keitaro repositories because of the openchain requirements. I just added a LICENSE file for the license that you indicated in the header of your source file, and I added a copyright notice, attributing the copyright to Keitaro.

### CODE REVIEW

File: remove-extra-feeds.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
   Author URI:  https://github.com/gocemitevski
   License:     GPL2
   License URI: https://www.gnu.org/licenses/gpl-2.0.html
+  Copyright:   Copyright (c) 2016 Keitaro AB
```
Comment (gocemitevski) [inline remove-extra-feeds.php]: Adding Copyright in that way will probably result with an invalid plugin header. https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields

There is an alternative way to add the Copyright value by specifying a PHPDoc DocBlock. For example:

```
<?php
/**
 * Plugin Name
 *
 * @package     PluginPackage
 * @author      Your Name
 * @copyright   2019 Your Name or Company Name
 * @license     GPL-2.0-or-later
 *
 * @wordpress-plugin
 * Plugin Name: Plugin Name
 * Plugin URI:  https://example.com/plugin-name
 * Description: Description of the plugin.
 * Version:     1.0.0
 * Author:      Your Name
 * Author URI:  https://example.com
 * Text Domain: plugin-slug
 * License:     GPL v2 or later
 * License URI: http://www.gnu.org/licenses/gpl-2.0.txt
 */
```

**Comment by ElenaKusevska** [comment]
@gocemitevski thank you for the reply. I'll add the DocBlock like you suggested.

**Comment by ElenaKusevska** [comment]
@gocemitevski please review the changes, and if it's OK merge the PR. If not, let me know if there is anything else I have to correct.

**Review by gocemitevski — APPROVED** [review]
Looks good to me.

**Pull request merged**